containers/jupyter-datascience-notebooks/.devcontainer/devcontainer.json

```
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json.
+{
+	"name": "Jupyter Data Science Notebooks",
```

```suggestion
	"name": "Jupyter Data Science Notebooks (Community)",
```
Would you mind adding a (Community) label here?